src/main/java/org/eclipse/cargotracker/interfaces/handling/rest/HandlingReport.java

```diff
@@ -8,35 +8,23 @@
 @XmlRootElement
 public class HandlingReport {
 
-    // TODO [Jakarta EE 8] Use not blank instead.
-    @NotNull(message = "Missing completion time.")
-    @Size(min = 16, max = 16, message = "Completion time value must be sixteen characters long.")
-    // TODO [DDD] Apply regular expression validation.
+    @NotNull
+    @Size(min = 16, max = 16)
     private String completionTime;
 
-    // TODO [Jakarta EE 8] Use not blank instead.
-    @NotNull(message = "Missing tracking ID.")
-    @Size(min = 4, message = "Tracking ID must be at least four characters.")
+    @NotNull
+    @Size(min = 4)
     private String trackingId;
 
-    // TODO [Jakarta EE 8] Use not blank instead.
-    @NotNull(message = "Missing event type.")
-    @Size(
-            min = 4,
-            max = 7,
-            message = "Event type value must be one of: RECEIVE, LOAD, UNLOAD, CUSTOMS, CLAIM")
-    // TODO [DDD] Apply regular expression validation.
+    @NotNull
```

Comment:
This is a sync mismatch. Can you kindly restore the more updated code?

Reply:
update.